core/drand_daemon.go

```diff
@@ -215,6 +242,9 @@ func (dd *DrandDaemon) LoadBeacons(metricsFlag string) error {
 		} else {
 			fmt.Printf("beacon id [%s]: will start running randomness beacon.\n", beaconID)
 
+			// Add beacon chain hash as a new valid one
+			dd.AddNewChainHash(beaconID, bp)
```

Comment:
does this mean we can have a race condition where the hash is set, but the beacon is not?